### PR TITLE
Update istio/istio ref and reenable tests

### PR DESCRIPTION
**Pull request by ericvn**
Update the istio/istio reference to most recent. This should fix the current issue with the failing tests where they are using the `latest` proxyv2 image which then fails to start.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: bin/init.sh [hunk 1]
```diff
@@ -48,8 +48,15 @@ git fetch "$ISTIO_REMOTE"
 git checkout "$ISTIO_SHA"
 
 # Build and install istioctl
-make gen-charts
-go install ./istioctl/cmd/istioctl
+export TAG=${ISTIO_IMAGE_VERSION}.$(git rev-parse ${ISTIO_SHA})
```
Comment (frankbu) [inline bin/init.sh:51]:
```suggestion
export TAG=${ISTIO_IMAGE_VERSION}.$(git rev-parse "${ISTIO_SHA}")
```

File: bin/init.sh [hunk 2]
```diff
@@ -48,8 +48,15 @@ git fetch "$ISTIO_REMOTE"
...
+export VERSION=${TAG}
+export ISTIO_VERSION=${TAG}
+echo "TAG=${TAG}"
+echo "VERSION=${VERSION}"
+echo "ISTIO_VERSION=${ISTIO_VERSION}"
+make gen-charts ${ISTIO_OUT}/release/istioctl-linux-amd64
+cp -a ${ISTIO_OUT}/release/istioctl-linux-amd64 /gobin/istioctl
```
Comment (frankbu) [inline bin/init.sh:58]:
```suggestion
make gen-charts "${ISTIO_OUT}/release/istioctl-linux-amd64"
cp -a "${ISTIO_OUT}/release/istioctl-linux-amd64" /gobin/istioctl
```

**Comment by ericvn** [comment]
@googlebot I consent.

**Comment by ericvn** [comment]
@googlebot I consent.

File: go.mod [hunk 3]
```diff
@@ -21,3 +21,5 @@ replace github.com/Azure/go-autorest/autorest => github.com/Azure/go-autorest/au
 replace github.com/Azure/go-autorest/autorest/adal => github.com/Azure/go-autorest/autorest/adal v0.5.0
 
 replace github.com/Azure/go-autorest => github.com/Azure/go-autorest v13.2.0+incompatible
+
+replace sigs.k8s.io/service-apis => github.com/howardjohn/service-apis v0.0.0-20200625215450-120bc372d5a7
```
Comment (howardjohn) [inline go.mod:25]: oof sorry, this is fixed now

**Pull request merged**